code/WorkInProgress/JobXPRewards.dm

```diff
@@ -482,3 +482,53 @@ mob/verb/checkrewards()
 		banana.set_loc(get_turf(C.mob))
 		C.mob.put_in_hand(banana)
 		return
+
+/////////////Bartender////////////////
+
+/datum/jobXpReward/bartender
+	name = "Golden Cocktail Shaker"
+	desc = "After all your years of service, you've finally managed to gather enough money in tips to buy yourself a present! You regret every cent."
+	required_levels = list("Bartender"=0)
```

Comment:
level 0 intentional?

Reply:
Testing purposes only, will change to 20 for final revision